lightwood/model/neural.py

```diff
@@ -184,7 +184,9 @@ def _max_fit(self, train_dl, dev_dl, criterion, optimizer, scaler, stop_after, r
 
             train_error = np.mean(running_losses)
 
-            running_errors.append(self._error(dev_dl, criterion))
+            epoch_error = self._error(dev_dl, criterion)
+            running_errors.append(epoch_error)
+            log.info(f'Loss @ epoch {epoch}: {epoch_error}')
```

Comment:
This will log too often in some cases, I think, add as `debug` if you think it's useful so we can disable it in mindsdb